src/app/ui/shared/components/atoms/card-study/card-study.component.html

```diff
@@ -32,6 +32,6 @@ <h3>
   }
 
   <button (click)="onCardClick()">
-    <span i18n>Go to study {{ title() }}</span>
+    <span><span i18n>Go to study</span> {{ title() }}</span>
```

Comment:
```suggestion
    <span><ng-container i18n>Go to study</ng-container> {{ title() }}</span>
```

Both span could be ng-containers if unused for design